### DISCUSSION TITLE
Execute multi process jobs via python

### DISCUSSION THREAD

**Discussion by bendnorman** [dagster-io/dagster#16176] [Q&A] [answered]
In PUDL, we use pytest to run our dagster ETL on a subset of the data. We're currently using the in-process executor but we'd like to take advantage of some parallelism and execute the job using a multi-process executor via python:

https://github.com/catalyst-cooperative/pudl/blob/bfcf49530980d26856c34fd417fa0d4e9f24c946/test/conftest.py#L329-L367

What is the best way to do this? Should we be using `build_reconstructable_job`?

**Comment by sryza** [comment, marked as answer]
Hey Bennett - take a look at https://github.com/dagster-io/dagster/discussions/8467

**Reply by bendnorman** [reply]
Thank you!
